src/Symfony/Component/VarDumper/Server/DumpServer.php

```diff
@@ -49,33 +55,18 @@ public function start(): void
         }
     }
 
-    public function listen(callable $callback): void
+    public function listen(callable $callback, ?StreamableInputInterface $streamInput): void
     {
-        if (null === $this->socket) {
+        if ($streamInput instanceof StreamableInputInterface && $stream = $streamInput->getStream()) {
+            $this->inputStream = $stream;
```

Comment:
it'd be better to not use a property to pass the stream to getMessage. Can we use an argument instead?

Reply:
Good idea.